tests/suites/test_suite_psa_crypto_metadata.function

```diff
@@ -56,17 +56,15 @@
 #define TEST_CLASSIFICATION_MACRO( flag, alg, flags )           \
     TEST_ASSERT( PSA_##flag( alg ) == !! ( ( flags ) & flag ) )
 
-/* Check the parity of value.
- * Return 0 if value has even parity and a nonzero value otherwise. */
-int test_parity( uint32_t value )
+/* Check if the parity of value is even.
```

Comment:
Please comment the _why_ in preference to the _what_. The job of this function is to test whether the parity is whichever parity is expected for numerical values. The fact that it's even parity is a detail.

See the discussion in https://github.com/ARMmbed/mbed-crypto/pull/330#discussion_r360305364 . This discussion also highlights that “even parity” is not such good documentation: Jaeden calls it “odd parity”!

My bad for not documenting why we're checking the parity of some values. Here's some proposed documentation.

```
/* Check the parity of value.
 * Numerical encodings of key types and related values such as EC curves and
 * DH groups should all have the same parity. This guarantees that a
 * single-bit error will be detected.
 * The expected parity is even because this makes all-bits-zero valid.
 */
int test_parity( uint32_t value )
```

Reply:
I've changed the function documentation taking into account your proposal. Please have a look to the new version.